src/freenas/usr/local/bin/truenas-grub.py

```diff
@@ -16,12 +16,12 @@ def get_serial_ports():
     advanced = query_config_table("system_advanced", prefix="adv_")
     kernel_extra_options = advanced.get("kernel_extra_options") or ""
 
-    # check for /data/.vendor
+    vendor = "TrueNAS Scale"
```

Comment:
Let's define these in enum. See conversation and coordinate with https://github.com/truenas/middleware/pull/14061

Reply:
Sounds good! I'll keep tabs on it

Reply:
I'll update this branch once that PR gets merged!